bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/LoggingBigQueryStorageReadRowsTracer.java

```diff
@@ -116,16 +118,18 @@ private static long perSecond(DurationTimer timer, long metric) {
   }
 
   private void logData() {
-    log.info(
-        "{}: Started: {} Ended: {} Parse Timings: {}  Time in Spark: {} Time waiting for service: {} Bytes/s: {} Rows/s: {}",
-        streamName,
-        startTime,
-        endTime == null ? "" : endTime.toString(),
-        format(parseTime),
-        difference(sparkTime, parseTime),
-        format(serviceTime),
-        perSecond(serviceTime, bytes),
-        perSecond(parseTime, rows));
+    JsonObject jsonObject = new JsonObject();
+    jsonObject.addProperty("Stream Name", streamName);
+    jsonObject.addProperty("Started", startTime == null ? "" : startTime.toString());
+    jsonObject.addProperty("Ended", endTime == null ? "" : endTime.toString());
+    jsonObject.addProperty("Parse Timings", format(parseTime));
+    jsonObject.addProperty("Time in Spark", difference(sparkTime, parseTime));
+    jsonObject.addProperty("Time waiting for service", format(serviceTime));
+    jsonObject.addProperty("Bytes/s", perSecond(serviceTime, bytes));
+    jsonObject.addProperty("Rows/s", perSecond(parseTime, rows));
+    jsonObject.addProperty("Bytes", bytes);
+    jsonObject.addProperty("Rows", rows);
+    log.info("Tracer Logs:{}", new Gson().toJson(jsonObject));
```

Comment:
Is similar change required for create read session log as well?
bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/ReadSessionCreator.java : 150

Reply:
Yeah it is Nilesh, but this change is mainly to get total bytes read and records read. 
Will work on ReadSessionCreator (to get the scan time) soon and raise another PR for it.